src/joy.c

```diff
@@ -983,45 +981,42 @@ static int process_multiple_input_files (joy_ctx_data *ctx, char *input_filename
             }
 #endif
         }
-    }
 
-    /* copy input filename path */
-    strncpy_s(input_path, (MAX_FILENAME_LEN-1), input_filename, (MAX_FILENAME_LEN-1));
+        /* copy input filename path */
+        strncpy_s(input_path, (MAX_FILENAME_LEN-1), input_filename, (MAX_FILENAME_LEN-1));
```

Comment:
This will create safec constraint message when using SAFEC. Src Len must be less than dst Len

Reply:
dos it generate a message now? because this code isn't new - just moving it inside an if condition.

Reply:
It should.  When running I would have to look for the message though. Does our current tests run this code?

Reply:
yes, if you run joy and specify as input a directory of pcap files
 there are also other places where the lengths are the same in the code.